pyproject.toml

```diff
@@ -1,8 +1,8 @@
 [tool.poetry]
 name = "librelane"
-version = "2.4.0.dev6"
+version = "2.4.1"
 description = "An infrastructure for implementing chip design flows"
-authors = ["Mohamed Gaber <me@donn.website>", "Efabless Corporation"]
+authors = ["Mohamed Gaber <me@donn.website>"]
```

Comment:
Not sure if we're legally allowed to do that. Efabless has to stay here for copyright purposes as they still own most of the codebase. This may change in the future, but not currently.

Reply:
That's right, Efabless is the copyright holder for most of the code that is already there.
But I'm pretty sure this is not a copyright notice. If so, you'd have to add Sylvain, Uri, me... to this list.
The actual copyright holders are listed in the source files with the keyword "Copyright", that's the legal requirement. (You could also do a blanket copyright notice over the whole project or parts of it, but more granular is better.)

The authors in pyproject.toml are the active authors of this project I would say, and that's you.
Keeping Efabless there would send the wrong signals, as they are not active authors of LibreLane.

Reply:
So `maintainers` is allowed (as well as many other pyproject.toml fields) with poetry 2.0.0 but that's Python 3.10+ exclusive. I think that's a fair sacrifice for LibreLane 3.0.0. For now I'll add a comment.